samples/spring-eventsourced-counter/src/main/java/com/example/actions/CounterJournalToTopicAction.java

```diff
@@ -0,0 +1,94 @@
+package com.example.actions;
+
+// tag::sub-ESE-pub-topic-action[]
+// tag::sub-ESE-action[]
+// tag::sub-topic-action[]
+import kalix.javasdk.action.Action;
+// end::sub-ESE-pub-topic-action[]
+// end::sub-ESE-action[]
+// end::sub-topic-action[]
+
+// tag::sub-ESE-pub-topic-action[]
+import kalix.springsdk.annotations.Publish;
+// end::sub-ESE-pub-topic-action[]
+
+// tag::sub-ESE-pub-topic-action[]
+// tag::sub-ESE-action[]
+// tag::sub-topic-action[]
+import kalix.springsdk.annotations.Subscribe;
+// end::sub-ESE-pub-topic-action[]
+// end::sub-ESE-action[]
+// end::sub-topic-action[]
```

Comment:
Maybe a easier way here is to use a specific tag for each import and then use that tag on docs.. when importing the file, you can pass in several tags... e.g. `CounterJournalToTopicAction.java[tag=importX,imporY,sub-ESE-pub-topic-action]`

Reply:
I didn't know that :) awesome! will do

Reply:
I tried and didn't seem to work. Do you have a sample of it? I'm looking at the Antora docs and can't find the proper info. There is some entry about 'Tags' but that's not what we are using here I think.

Reply:
My bad, correct form is `tags` and using `;` as separator.. so: CounterJournalToTopicAction.java[tags=importX;importY;sub-ESE-pub-topic-action]

See example in views.adoc#L46 from java [here](https://github.com/lightbend/kalix-jvm-sdk/blob/main/docs/src/modules/java/pages/views.adoc#L46)

Reply:
I had to divide it in two classes so I don't need it for now but I keep it in mind. It's very neat 👍